client/components/Cart.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { connect } from "react-redux";
+import { useHistory } from "react-router-dom";
+
+
+export const Cart = (props) => {
+  let history = useHistory();
+  const { username } = props;
```

Comment:
might want to change this to 'email' - kevin and i have or will both reference email rather than username in our code since we're no longer storing a username in the db. tbh tho, i'm not clear on where props are coming from in this case, so i could be off.

Reply:
Gotcha, at this point in the code, it wouldn't make a difference since it's not communicating with the backend, so I can just remove it entirely for now.

Reply:
all good. i think we're good to go then. i'll approve